WebApp.E2ETests/Pages/ExaminationsPage.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace WebApp.E2ETests.Pages
+{
+    public class ExaminationsPage
+    {
+        public ExaminationsPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        private readonly IWebDriver _driver;
+        private const string URI = "http://localhost:8080/#/examinations";
```

Comment:
For now this is alright, since it will only be run on local deployment, but it should be parametrized in the future.